scss/main.scss

```diff
@@ -528,7 +535,7 @@ h1,h2,h3 {
 }
 
 .volunteersRow {
-  margin-top: 64px;
+  margin-top: 80px;
```

Comment:
Seems like we now have 162px of space here on mobile.
Can you please adjust? There's also a few merge conflicts as well. 🎉

PR:

<img width="401" alt="Screen Shot 2021-02-18 at 9 13 02 AM" src="https://user-images.githubusercontent.com/9326234/108377384-978e9080-71c9-11eb-80fe-e694fa35b682.png">

Design:
<img width="208" alt="Screen Shot 2021-02-18 at 9 12 35 AM" src="https://user-images.githubusercontent.com/9326234/108377420-a07f6200-71c9-11eb-94db-bcdad19cf651.png">

Reply:
Okay, I got that fixed. As far as the merge conflicts though it _seems_ okay, but I'd feel a lot better if you double checked my work. I merged it locally since Github was saying the conflicts were in the CSS file not the SASS file (which I've never done before). If there is a problem maybe it would be easier to abandon this branch and just make a new one now that we have the designs with more information? Let me know. Thanks!

Reply:
Hmm..can you revert your `main.css` and `main.css.map` files back to what is on master?

Run `git checkout origin/master main.css` and `git checkout origin/master main.css.map` and commit.
No need to commit any changes to CSS files.